source/codegen/templates/service_helpers.mako

```diff
@@ -17,6 +18,7 @@
   session_field_name = next(explicit_session_params, 'session_name')
 %>\
 ${initialize_input_params(function_name, parameters)}
+${initialize_output_params(output_parameters_to_initialize)}\
```

Comment:
Since `Initialize()` returns an extra output parameter we need to initialize it.